default.nix

```diff
@@ -37,6 +37,10 @@ let
       chmod -R +w $out
       cd $out
       patch -p1 -i ${./nix/cardano-wallet--enable-aarch64-darwin.patch}
+      patch -p1 -i ${pkgs.fetchurl {
```

Comment:
Yup, although it was merged into `master`, I’m not sure if we should use branches that are not their releases. Otherwise, I could have just pointed to that PR commit directly, while here I’m only applying that PR’s diff onto an official release.

@danielmain suggests we merge `release/4.12.0` into `develop`, but that can only happen after Wednesday’s release, so here’s something you can merge into your own branch today, if you get bitten by the 1.35.0’s ledger bug =)

Reply:
I did indeed, I 🍒 -picked the commit though.

Reply:
Sure, whatever you prefer. =) Git will do the right thing, whether we merge `release/4.12.0` or this one, as long as file contents are the same

Reply:
The reason I was asking was, I wanted to raise an equivalent PR in our other repo and I was wondering if just changing hashes would be enough or should we actually do similar patching.

Reply:
Hopefully, they’ll release soon, and we’ll be able to clean this up =)